samples/client/petstore/php/OpenAPIClient-php/lib/Api/DefaultApi.php

```diff
@@ -156,21 +156,6 @@ public function fooGetWithHttpInfo()
                         $response->getHeaders()
                     ];
             }
```

Comment:
I would also remove the switch block as there is only a default case, no real condition